tests/test-recipes/track_feature_v1/meta.yaml

```
@@ -0,0 +1,8 @@
+package:
+  name: track_feature
+  version: 1.0

```

```suggestion
  version: "1.0"
```

In case you see something weird in the versioning.